acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/factory/PantheonFactoryConfiguration.java

```diff
@@ -40,7 +41,8 @@
       final PermissioningConfiguration permissioningConfiguration,
       final boolean devMode,
       final GenesisConfigProvider genesisConfigProvider,
-      final Boolean p2pEnabled) {
+      final Boolean p2pEnabled,
+      final boolean discovery) {
```

Comment:
Nitpick: Field that's being set is `Boolean`, this is `boolean`.  Should probably be the former.